web/src/engine/websites/Bomtoon.ts

```diff
@@ -37,4 +61,53 @@ export default class extends Delitoon {
         return results.distinct();
     }
 
+    public override async FetchPages(chapter: Chapter): Promise<Page<ScrambleParams>[]> {
+        await this.UpdateToken();
+        const url = new URL(`contents/viewer/${chapter.Parent.Identifier}/${chapter.Identifier}`, this.apiUrl);
+        url.searchParams.set('isNotLoginAdult', 'true');
+        const { result, data: { images, isScramble } } = await FetchJSON<APIResult<APIPages>>(this.CreateRequest(url));
+        if (result === 'ERROR') {
```

Comment:
Does the thrown error message really match this result condition?
I would expect a more specific result like `FORBIDDEN` to fit a message about login or purchased chapters.